lib/api/auth/authentication.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: use_build_context_synchronously
```

Comment:
I was actually thinking the same when going over the code. I'm aware one should always check if current widget is mounted and not disposed to prevent errors. I will do as suggested 💯